Jenkinsfile

```
@@ -66,23 +75,93 @@ pipeline {
       }
     }
 
-    stage('Run Tests') {
+    stage('Run environment tests in parallel') {
+      failFast true
       parallel {
-        stage('RSpec') {
-          steps { sh 'ci/test rspec' }
-        }
-        stage('Authenticators Config') {
-          steps { sh 'ci/test cucumber_authenticators_config' }
-        }
-        stage('Authenticators Status') {
-          steps { sh 'ci/test cucumber_authenticators_status' }
-        }
-        stage('LDAP Authenticator') {
-          steps { sh 'ci/test cucumber_authenticators_ldap' }
-        }
-        stage('OIDC Authenticator') {
-          steps { sh 'ci/test cucumber_authenticators_oidc' }
-        }
+          stage('EE FIPS agent tests') {
+            agent { label 'executor-v2-rhel-ee' }
+            when {
```

If you use getTrigger to determine if the build was started by a timer, do you still need parameterised cron and the NIGHTLY parameter?
```suggestion
            when { expression { return getTrigger() == "timer" } }
```

---

I would like to have the option to trigger the job manually and get EE suite tests running, to allow developers a quick tool for EE coverge testing, WDYT? 

Timer can be defined for different reasons, even on feature branch - for example running 30 builds by trigger should not invoke EE tests